src/dmd/dinterpret.d

```diff
@@ -160,7 +160,7 @@ public extern (C++) void printCtfePerformanceStats()
 /*********
  * Typesafe PIMPL idiom so we can keep CompiledCtfeFunction private.
  */
-struct CompiledCtfeFunctionPimpl
+extern (C++) struct CompiledCtfeFunctionPimpl
```

Comment:
This is an internal struct?  I think this can be ignored.

Reply:
This is because in `FuncDeclaration` we store a `CompiledCtfeFunctionPimpl` [member field](https://github.com/dlang/dmd/blob/master/src/dmd/func.d#L266) which was manually re-written in the `declaration.h` header file as `CompiledCtfeFunction *` ([link](https://github.com/dlang/dmd/blob/master/src/dmd/declaration.h#L509))

Reply:
I also feel that this should stay private, but this was the easiest solution I came up with. Open to suggestions

Reply:
Ahh... used in func.d, in a rather weird way when comparing against the existing C++ header.